README.md

```diff
@@ -1436,6 +1437,38 @@ nvim_lsp#setup("pyls", {config})
     settings = {}
 ```
 
+## pyls_ms
+
+https://github.com/Microsoft/python-language-server
+
+`Microsoft Language Server for Python `, a language server for Python.
+    
+Requires [.NET Core](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script) to run. On Linux or macOS:
+
+```bash
+curl -L https://dot.net/v1/dotnet-install.sh | sh
+```
```

Comment:
this got lost in the auto-update https://github.com/neovim/nvim-lsp/commit/aa187f1afb5ee1c7cf55089f8c8dc7da44c931c6 

I think it should go in the `docs` ?

Reply:
Ah sorry, I didn't read the docgen section. I can make a PR to fix this.